kernel/aster-nix/src/fs/file_table.rs

```diff
@@ -240,13 +279,30 @@ impl FileTableEntry {
             file,
             flags: AtomicU8::new(flags.bits()),
             subject: Subject::new(),
+            owner: RwLock::new(None),
         }
     }
 
     pub fn file(&self) -> &Arc<dyn FileLike> {
         &self.file
     }
 
+    pub fn owner(&self) -> Option<Pid> {
+        self.owner.read().as_ref().map(|(pid, _)| *pid)
+    }
+
+    pub fn set_owner(&self, owner: Pid) -> Result<()> {
```

Comment:
The meaning of this so-called owner desires documentation. You don't even know the meaning before you write this PR, right? Other people are no smarter than you. So please write down what you learn as comments for future readers, including a six-month-later you.